command/agent/command.go

```diff
@@ -472,10 +472,18 @@ func SetupLoggers(ui cli.Ui, config *Config) (*logutils.LevelFilter, *gatedwrite
 
 	// Create a log writer, and wrap a logOutput around it
 	writers := []io.Writer{logFilter}
-
+	logLevelMap := map[string]gsyslog.Priority{
+		"OFF":   gsyslog.LOG_EMERG,
```

Comment:
When the log level is `"OFF"` the user is saying they don't want logs at all (which is intended only for testing but it's an option). Maybe if `config.LogLevel` is `"OFF"` we can treat that exactly as if `config.EnableSyslog` is false?

Reply:
Yes. It makes sense to me too. Let me update the code